public/index.html

```diff
@@ -1267,6 +1267,37 @@ <h4 class="wide100p textAlignCenter">
                                             </div>
                                         </div>
                                     </div>
+                                    <div data-newbie-hidden data-tg-type="mancer, ooba, koboldcpp, aphrodite, tabby" name="dryBlock" class="wide100p">
```

Comment:
This only applies to ooba, so leave just ooba in data-tg-type attribute

Reply:
[An implementation for llama.cpp](https://github.com/ggerganov/llama.cpp/pull/6839) is on the way!

Reply:
Still not merged tho. So we wait again.